test_prelu fails frequently.

```
======================================================================

FAIL: test_operator.test_prelu

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/usr/lib/python3.6/site-packages/nose/case.py", line 198, in runTest

    self.test(*self.arg)

  File "/work/mxnet/tests/python/unittest/common.py", line 172, in test_new

    orig_test(*args, **kwargs)

  File "/work/mxnet/tests/python/unittest/test_operator.py", line 827, in test_prelu

    check_symbolic_forward(y, [xa, gam_full], [ya_full], rtol=rtol, atol=atol, dtype=dtype)

  File "/work/mxnet/python/mxnet/test_utils.py", line 997, in check_symbolic_forward

    equal_nan=equal_nan)

  File "/work/mxnet/python/mxnet/test_utils.py", line 491, in assert_almost_equal

    raise AssertionError(msg)

AssertionError: 

Items are not equal:

Error 100.000000 exceeds tolerance rtol=0.010000, atol=0.001000.  Location of maximum error:(0, 0, 2, 2), a=-0.045013, b=117.000000

 EXPECTED_leakyrelu24_output: array([[[[-0.0745  ,  0.3264  ,  0.7534  , -0.0559  , -0.05847 ],

         [-0.0293  , -0.002966, -0.05618 , -0.0824  , -0.016   ],

         [-0.05652 , -0.0627  , -0.045   , -0.013756,  0.8813  ],...

 FORWARD_leakyrelu24_output: array([[[[ -0.0745  ,   0.3264  ,   0.7534  ,  -0.0559  ,  -0.05847 ],

         [ -0.0293  ,  -0.002966,  -0.05618 ,  -0.0824  ,  -0.016   ],

         [ -0.05652 ,  -0.0627  , 117.      ,   8.164   ,   0.8813  ],...

-------------------- >> begin captured logging << --------------------

common: INFO: Setting test np/mx/python random seeds, use MXNET_TEST_SEED=409746395 to reproduce.

--------------------- >> end captured logging << ---------------------
```

http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-12810/8/pipeline/1005
http://jenkins.mxnet-ci.amazon-ml.com/blue/organizations/jenkins/incubator-mxnet/detail/PR-12810/7/pipeline

--- a/tests/python/unittest/test_operator.py
+++ b/tests/python/unittest/test_operator.py
@@ -794,7 +794,6 @@ def test_leaky_relu():
 # NOTE(haojin2): Skipping the numeric check tests for float16 data type due to precision issues,
 # the analytical checks are still performed on each and every data type to verify the correctness.
 @with_seed()
-@unittest.skip("Flaky test tracked by https://github.com/apache/incubator-mxnet/issues/12885")
 def test_prelu():
     def fprelu(x, gamma):
         pos_indices = x > 0

